_layouts/default.html

```diff
@@ -45,7 +46,7 @@ <h2 class="project-tagline">{{ page.description | default: site.description | de
     </header>
 
     <main id="content" class="main-content" role="main">
-      {% include anchor_headings.html html=content anchorBody="§" anchorClass="heading-anchor" anchorAttrs="aria-labelledby=\"%html_id%\"" headerAttrs="class=heading" %}
+      {% include anchor_headings.html html=content anchorClass="heading-anchor fa fa-anchor" anchorAttrs="aria-labelledby=\"%html_id%\"" headerAttrs="class=heading" %}
```

Comment:
I suggest to use `fa-link` symbol
![image](https://user-images.githubusercontent.com/502695/144870831-4579af9b-8f34-42c9-9fdb-c23f0ae13b3e.png)

instead of `fa-anchor`
![image](https://user-images.githubusercontent.com/502695/144870989-d94925f2-a698-47e4-a712-cc0f1d8c2e90.png)

Most documentation-centric themes and CMS are using "link" symbol for header anchors and it became a de-facto standard recognizable by users.

Reply:
well, both are better than `§`. I like the "anchor" anchor better b/c it's aligned w/ the k8s nomenclature but I guess we can be ~boring~ predictable in our docs ;) so changing..